src/scripts/pokemons/pokemon-list.js

```diff
@@ -12,39 +13,39 @@ class PokemonList {
       const newPoke = new Pokemon(name, 0, reference);
       this.add(newPoke);
       newPoke.onLike.addActions(() => apiMicroverse.setLike(name));
+      newPoke.onOpenComments.addActions(() => modal.openComments(newPoke));
     });
-    this.loadLikes();
-    this.loadImages();
+    this.loadData();
   }
 
   add = (pokemon) => {
     this.pokemons.push(pokemon);
   }
 
-  loadLikes = () => {
-    this.pokemons.forEach((poke) => {
-      this.setLike(poke);
-    });
-  }
-
   setLike = async (poke) => {
     const extLikes = await apiMicroverse.getLikes(poke.name);
     poke.setLike(extLikes);
   }
 
-  getImage = async (name) => {
+  // Request a pokeApi
+
+  getData = async (name) => {
     const response = await access.getApi(`${routes.POKEMON}${name}`, {});
-    return response.sprites.front_default;
+    return response;
   }
 
-  setImage = async (element) => {
-    const img = await this.getImage(element.reference.querySelector('.pokemon-name').innerText);
-    element.setImage(img);
+  // method to get data from the pokeApi, update as needed. (YAGNI)
```

Comment:
```suggestion
  // method to get data from the pokeApi, update as needed. (YAGNI)
```

- [ ] Kindly remove this comments